entrypoint.sh

```diff
@@ -30,3 +30,15 @@ echo "::add-mask::$LAZY_API_URL"
   echo "LAZY_API_URL=$LAZY_API_URL"
 } >>"$GITHUB_ENV"
 
+echo "::add-mask::$GITHUB_ID"
+{
+  echo "GITHUB_ID=$GITHUB_ID"
```

Comment:
Did you mean GITHUB_APP_ID?

Reply:
good catch, also added GITHUB_REPOSITORY